actions/pack_action.go

```diff
@@ -30,12 +34,43 @@ type PackAction struct {
 	File             string
 }
 
+func (pf *PackAction) Verify(context *debos.DebosContext) error {
+
+	if len(pf.Compression) > 0 {
+		possibleTypes := []string{"gz", "bzip2", "xz", "none"}
+		for _, a := range possibleTypes {
+			if a == pf.Compression {
+				return nil
+			}
+		}
+
+		return fmt.Errorf("Option 'compression' has an unsupported type: `%s`. Possible types are %s.",
+			pf.Compression, strings.Join(possibleTypes, ", "))
+	}
+
+	return nil
+}
+
 func (pf *PackAction) Run(context *debos.DebosContext) error {
 	pf.LogStart()
 	outfile := path.Join(context.Artifactdir, pf.File)
 
+	var tarOpt = "cf"
+	if len(pf.Compression) > 0 {
```

Comment:
makes it more readable if you set the default compression type rather than the tar options; also this probably belongs in a constructor function like https://github.com/go-debos/debos/blob/master/actions/debootstrap_action.go#L77